### ISSUE TITLE
Login fails since mid April

### ISSUE THREAD

**Issue by enigmawelt** [EmDee/moneymoney-bondora#23]
Other apps such as Outbank have also had problems since mid-April. I have been in contact with the support there since then and uploaded a network log (suffix HAR) to the support today. According to Bondora support, nothing has been changed in their own API, but I wonder if Outbank and this extension are using the API at all.

**Comment by EmDee** [comment]
@enigmawelt Have you tried updating the Bondora extension? Bondora changed their API endpoint, see https://github.com/EmDee/moneymoney-bondora/issues/22.

**Comment by enigmawelt** [comment]
I had only looked at the homepage of the Git project and didn't notice any changes there, thanks.

**Comment by EmDee** [comment]
@enigmawelt You can either download the Bondora.lua (it is signed by MoneyMoney)from the release page or get the extension directly from the MoneyMoney website. Both are linked in the other issue.

**Comment by enigmawelt** [comment]
Hallo Christine,

Schön stand nicht dabei, sieht man auch im Video was ich gemacht habe.

Die Extension von MoneyMoney ist mitlerweile gefixt, hier der Link zum Git

https://github.com/EmDee/moneymoney-bondora￼
EmDee/moneymoney-bondora: MoneyMoney extension for Bondora
github.com

Eventuell findet man darin schon die Änderungen.

VG

**Comment by mdohnke** [comment]
I still have the problem that the extension is not working (current version downloaded from MoneyMoney Release page). 
Log attached.
[bondora.log](https://github.com/user-attachments/files/15924286/bondora.log)

**Comment by mdohnke** [comment]
I developed an new extension based on OAuth2 and the Bondora API. You can find it here: https://github.com/mdohnke/moneymoney-bondora-oauth

**Comment by EmDee** [comment]
@mdohnke This is great! I think it makes sense that you write MM to get officially listed on their extensions page. Maybe add (oAuth) or similar in the name to prevent confusion and compatibility issues with this extension. And then I'll have this extension removed from their page.

**Comment by mdohnke** [comment]
Yes, I wrote them already and I added OAuth2 in the bank code, so directly in MM there should be no confusion. I will append oauth to the file itself, so both can exist in parallel.
